README.md

```diff
@@ -8,30 +8,40 @@
 
 spark-etl is a Scala-based project and it is developing with Spark. So it is scalable and distributed. spark-etl will process data from N source to N database.
 The project structure:
-- Extract
+
+**Extract**
+
+<img src="https://media.giphy.com/media/26BRIuiQfAxf5cENO/giphy.gif" alt="alt text" width="250" height="250">
```

Comment:
Ah, sorry. You just cannot resize images.
There is a notation like `![extract](https://media.giphy.com/media/26BRIuiQfAxf5cENO/giphy.gif =250x250)`
but I couldn't managed to make it work